app/boards/shields/jiran/jiran.dtsi

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2021 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <dt-bindings/zmk/matrix_transform.h>
+
+/ {
+	chosen {
+		zmk,kscan = &kscan0;
+		zmk,matrix_transform = &default_transform;
+	};
+
+	default_transform: keymap_transform_0 {
+		compatible = "zmk,matrix-transform";
+		columns = <12>;
+		rows = <5>;
+
+			// | SW1  | SW2  | SW3  | SW4  | SW5  | SW6  |   | SW6  | SW5  | SW4  | SW3  | SW2  | SW1  |
+			// | SW7  | SW8  | SW9  | SW10 | SW11 | SW12 |   | SW12 | SW11 | SW10 | SW9  | SW8  | SW7  |
+			// | SW13 | SW14 | SW15 | SW16 | SW17 | SW18 |   | SW18 | SW17 | SW16 | SW15 | SW14 | SW13 |
+			//                      | SW19 | SW20 | SW21 |   | SW21 | SW20 | SW19 |
```

Comment:
It looks like this is the wrong shape compared to the actual transform. This has 42 keys whereas the map has 56. The same goes for the other ones. Also I'm thinking the maps should be aligned using spaces, as the tabs result in something I assume wasn't intended on GitHub

Reply:
Oh, I'll fix that up in a second, sorry.

Reply:
I initially copied these in when I was editing the shield from the Jian back in March, totally forgot to fix them up for the PR. Done!